backend/BjjAppApi/Services/MovesMongoRepository.cs

```diff
@@ -1,13 +1,14 @@
+using BjjAppApi.Interfaces;
 using BjjAppApi.Models;
 using MongoDB.Driver;
 
 namespace BjjAppApi.Services;
 
-public class MovesService
+public class MovesMongoRepository : IMovesRepository
 {
     private readonly IMongoCollection<Move> _movesCollection;
 
-    public MovesService(IMongoCollection<Move> movesCollection)
+    public MovesMongoRepository(IMongoCollection<Move> movesCollection)
```

Comment:
I would imagine you would also need to pass the `IMongoCollection<Demo>` object into the constructor as well right? Since you need to provide the demo data along with the moves.

Reply:
Yup but that's the next thing I'm going to be working on. I wanted to implement the repository pattern first assuming that each model has its own collection.